requirements-minimal.txt

```diff
@@ -22,5 +22,6 @@ PyYAML>=5.1
 requests
 task_processing[mesos_executor,k8s]
 Twisted>=19.7.0
+typing-extensions>=3.10.0.0
```

Comment:
```suggestion
```

this line shouldn't be here since tron doesn't directly import this - we have a bit of a funky requirements setup here since some of our dependencies aren't public, but the typing_extensions dependency should be coming from task_processing which is a public dependency

can you share (either here or in slack) the error you're seeing without this line?